index.js

```diff
@@ -59,24 +55,24 @@ module.exports = customService(async function index(service) {
   service.decorate('GREETING_TYPE', {
     HELLO: 'hello',
   })
-  service.addRawCustomPlugin('POST', '/greetings/:userId', async function handler(req, reply) {
-    const currentUserId = req.params.userId
-    if (!currentUserId) {
-      throw new Error('This API should be consumed only by authenticated users!')
-    }
-    const userToSayHello = req.body.who
+  service.addRawCustomPlugin('POST', '/greetings', async function handler(req, reply) {
+    const userSayingHello = req.body.from
```

Comment:
There's no problem on how you've implemented the handler but You could have used object destructuring:

e.g.
`const {from, to} = req.body`

Reply:
Thanks for the suggestion! I have applied it! Now the code looks definitely nicer!